packages/utils/src/types.ts

```diff
@@ -216,6 +216,10 @@ export interface TemplatesType<T = any, F = any> {
   TitleFieldTemplate: React.ComponentType<TitleFieldProps<T, F>>;
   /** The template to use for rendering information about an unsupported field type in the schema */
   UnsupportedFieldTemplate: React.ComponentType<UnsupportedFieldProps<T, F>>;
+  /** The template to use for rendering a field that allows a user to add additional properties */
+  WrapIfAdditionalTemplate: React.ComponentType<
+    WrapIfAdditionalTemplateProps<T, F>
```

Comment:
Can you update the `getTemplate()` tests to add a this new template into the two places where the `registry` is loading test `templates`